recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
 
 test:
     requires:
-        - python {{ environ['PY_VER'] + '*' }}  # [win]
+        - python {{ PY_VER ~ '*' }}  # [win]
```

Comment:
Another fix. `environ.get('PY_VER')`.

I'd go for the latter until the conda-build fix is available to us.